packages/core/src/shared/extensionGlobals.ts

```diff
@@ -191,7 +191,7 @@ export interface ToolkitGlobals {
     /**
      * Log messages. Use `outputChannel` for application messages.
      */
-    logOutputChannel: OutputChannel
+    logOutputChannel: LogOutputChannel
```

Comment:
`LogOutputChannel` extends `OutputChannel` and includes the ability to add listeners.